### PR TITLE
#16 interactive slider

### LINKED ISSUE #16: Create interactive slider

### PR DESCRIPTION
**Pull request by Adrian-Mag**

### CODE REVIEW

File: climpred/model.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 class Model():
     """ Docs
     """
 
     def __init__(self):
-
-        self.previous_value = ''
-        self.value = ''
-        self.operator = ''
+        plot_obj = cp.Plot([1, 2, 3])
```
Comment (justinleung4732) [inline climpred/model.py:9]: Will need to change these values so that they match the initial slider inputs

**Review by justinleung4732 — APPROVED** [review]

File: climpred/__init__.py [hunk 2]
```diff
@@ -13,3 +13,4 @@
 from .controller import Controller  # noqa
 from .view import View  # noqa
 from .plot import Plot # noqa
+from .temperature_profile_calculation import calculate_temperature_matrix # noqa
```
Comment (sscivier) [inline climpred/__init__.py]: add one blank line at the bottom (flake8)

File: climpred/controller.py [hunk 3]
```diff
@@ -13,6 +13,14 @@ def __init__(self):
     def main(self):
         self.view.main()
 
+    def _on_slider_slide(self, event):
+        input_value = self.view.slider.get()
+        input_value_overwritten = cp.calculate_temperature_matrix()
```
Comment (sscivier) [inline climpred/controller.py:21]: Add comment saying something about how you're using dummy parameters

File: climpred/temperature_profile_calculation.py [hunk 4]
```diff
@@ -63,4 +63,4 @@ def calculate_temperature_matrix():
     epsilon_inverse_matrix = np.linalg.inv(epsilon_matrix)
 
     temperature_matrix = (np.dot(epsilon_inverse_matrix, rhs_vector))**0.25
-    return np.around(temperature_matrix, 2)
+    return np.around(temperature_matrix - 273.15, 2)
```
Comment (sscivier) [inline climpred/temperature_profile_calculation.py:67]: Add comment to specify that you're converting to celsius, and that we'll add functionality to change later

File: climpred/model.py (hunk 1 above)
Comment (shirin-ermis) [reply climpred/model.py:9]: Is the input here the temperature values that come out of the maths?

**Review by sscivier — CHANGES_REQUESTED** [review]
I can't really comment on the tkinter parts. Some minor changes.

**Review by sscivier — APPROVED** [review]

File: climpred/__init__.py (hunk 2 above)
Comment (justinleung4732) [inline climpred/__init__.py:16]: add "from .albedo import get_albedo

**Pull request merged**